content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -0,0 +1,280 @@
+ansible --version # Ansible version 2.4.x is required.
+```
+<br />Configure SODA hotpot install variables:
+
+You need to modify host_ip in `group_vars/common.yml`( Modify host_ip and change it to the actual machine IP of the host.), You can also specify which projects to deploy; only hotpot, only gelato or all.
+<br />
+
+```bash
+# This field indicates local machine host ip
+host_ip: 127.0.0.1
+# This field indicates which project should be deploy
+# 'hotpot', 'gelato' or 'all'
+deploy_project: all  #all refers to hotpot + gelato
+```
+`If you want to integrate SODA hotpot with k8s csi, please modify nbp_plugin_type to csi in group_vars/sushi.yml:`
+```bash
+# 'hotpot_only' is the default integration way, but you can change it to 'csi'
+# or 'flexvolume'
+nbp_plugin_type: hotpot_only
+```
+<Strong> LVM</Strong>
+<br/>
+
+If lvm is chosen as storage backend, modify `group_vars/osdsdock.yml:`<br />
+
+```bash
+enabled_backends: lvm
+```
+<Strong> NFS</Strong>
+<br/>
+
+If nfs is chosen as storage backend, modify `group_vars/osdsdock.yml:`<br/>
+
+```bash
+enabled_backends: nfs
+```
+<Strong>ceph</Strong>
+<br/>
+
+If ceph is chosen as storage backend, modify `group_vars/osdsdock.yml:`<br/>
+
+```bash
+enabled_backends: ceph # Change it according to the chosen backend. Supported backends include 'lvm', 'ceph', and 'cinder'.
+```
+<br />
+
+Configure `group_vars/ceph/all.yml` with an example below:<br />
+
+```bash
+ceph_origin: repository
+ceph_repository: community
+ceph_stable_release: luminous # Choose luminous as default version
+public_network: "192.168.3.0/24" # Run 'ip -4 address' to check the ip address
+cluster_network: "{{ public_network }}"
+monitor_interface: eth1 # Change to the network interface on the target machine
+devices: # For ceph devices, append ONE or MULTIPLE devices like the example below:
+  #- '/dev/sda' # Ensure this device exists and available if ceph is chosen
+  #- '/dev/sdb'  # Ensure this device exists and available if ceph is chosen
+osd_scenario: collocated
+```
+<Strong> cinder</Strong>
+<br />
+
+If cinder is chosen as storage backend, modify `group_vars/osdsdock.yml:`<br />
+
+```bash
+enabled_backends: cinder # Change it according to the chosen backend. Supported backends include 'lvm', 'ceph', and 'cinder'
+
+# Use block-box install cinder_standalone if true, see details in:
+use_cinder_standalone: true
+```
+<br />
+
+Configure the auth and pool options to access cinder in `group_vars/cinder/cinder.yaml`. Do not need to make additional configure changes if using cinder standalone.<br />
+<Strong>How to enable Telemetry installation</Strong>
+<strong>NOTE</Strong> : Please ensure that you are using hotpot version >= v0.6.1.
+<br />
+
+Update the file `ansible/group_vars/telemetry.yml` and change the value of enable_telemetry_tools to true
+
+```bash
+# Do you need to install or clean up telemetry tools?
+enable_telemetry_tools: false
+```
+<br /> How to enable Orchestration installation
```

Comment:
nit: These are subheadings. Please make them #### or ### depending on the nesting.